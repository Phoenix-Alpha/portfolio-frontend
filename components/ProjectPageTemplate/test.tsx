import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { ProjectPageTemplate } from '.'

describe('<ProjectPageTemplate />', () => {
  it('should renders correct <ProjectPageTemplate />', () => {
    const wrapper = shallow(<ProjectPageTemplate />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
