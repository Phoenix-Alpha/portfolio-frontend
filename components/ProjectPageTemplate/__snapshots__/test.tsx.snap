// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProjectPageTemplate /> should renders correct <ProjectPageTemplate /> 1`] = `
<Fragment>
  <ProjectSummaryListSection />
</Fragment>
`;
